infrastructure/index.ts

```diff
@@ -300,6 +300,19 @@ const distributionArgs: aws.cloudfront.DistributionArgs = {
     },
 };
 
+// domainAliases is a list of CNAMEs that accompany the CloudFront distribution. Any
+// domain name to be used to access the website must be listed here.
+const domainAliases = [];
+// websiteDomain is the A record for the website bucket associated with the website.
+domainAliases.push(config.websiteDomain);
+// targetDomain is the A record associated with the bucket populated by Pulumi. It may be
```

Comment:
Nit: "It may be removed that bucket is removed." seems like it is missing a word.